packages/shylu/shylu_dd/frosch/src/Adapters/Thyra_FROSchLinearOp_def.hpp

```diff
@@ -56,7 +56,9 @@ namespace Thyra {
     // Constructors/initializers
     template <class SC, class LO, class GO, class NO>
     FROSchLinearOp<SC,LO,GO,NO>::FROSchLinearOp()
-    {}
+    {
```

Comment:
See above (you don't need to define the constructor explicitly if its body would be empty anyway; either let the compiler define it or use the `=default` syntax in the class declaration).